delta/service/src/test/resources/akka-test.conf

```diff
@@ -0,0 +1,67 @@
+akka {
+  actor {
+    provider = "akka.cluster.ClusterActorRefProvider"
+    warn-about-java-serializer-usage = off
+
+    serializers {
+      kryo = "io.altoo.akka.serialization.kryo.KryoSerializer"
+    }
+
+    serialization-bindings {
+      "ch.epfl.bluebrain.nexus.delta.sdk.model.permissions.PermissionsCommand"   = "kryo"
```

Comment:
I'm not fond of the idea for two reasons:
- the sdk should not have any information about methods of serialisation
- one needs to control how serialisation happens (e.g. we want to serialise PermissionsEvent as json but the rest with kryo); adding a marker trait would make the SDK module responsible of the implementation, which would be incorrect.

Reply:
We can't have an Event trait and use it as a marker ?

And there is a resolution strategy when they are several candidates so we can still use a different serialization for Permissions: https://doc.akka.io/docs/akka/current/serialization.html?language=scala

Reply:
We can have a global Event trait, but it won't help much... because you'd have to have a serialiser for it that knows how to serialise instances of that type. So it would be part of the serialisation logic in one place, and the rest in another. Additionally, you'd still have to create a different one for the Command, State and Rejection... but this is bad for the reasons mentioned above.

The serialisation config is a map... so there's no resolution strategy for it...:
```
  serialization-bindings {
    "docs.serialization.JsonSerializable" = jackson-json
    "docs.serialization.CborSerializable" = jackson-cbor
    "com.google.protobuf.Message" = proto
    "docs.serialization.MyOwnSerializable" = myown
  }
```

Reply:
I was talking about this:
```You only need to specify the name of a trait or abstract base class of the messages. In case of ambiguity, i.e. the message implements several of the configured classes, the most specific configured class will be used, i.e. the one of which all other candidates are superclasses. If this condition cannot be met, because e.g. two marker traits that have been configured for serialization both apply and neither is a subtype of the other, a warning will be issued.```

Reply:
Are you suggesting to create two non-sealed traits, like so:
```
trait Message extends Product with Serializable
trait Event extends Message
```
... and mix them in all SDK types? (commands, states, rejections and events)

Reply:
If we modify sourcing to wrap the state in a case class, we can avoid having to mix in anything for commands, states and rejections.
And we can create a marker trait for Events.